src/Sulu/Bundle/AdminBundle/Resources/js/containers/CKEditor5/ckeditor5.scss

```diff
@@ -79,6 +80,17 @@ $textEditorUnpublishedLinkColor: $gold;
         .ck.ck-heading-dropdown .ck.ck-dropdown__button {
             border: 1px solid $textEditorDarkBorderColor;
         }
+
+        /**
+         * Move the sticky-toolbar below the SULU header.
+         * Still causes issues when snackbar is visible or when users scrolls whole page to top while editor with
+         * sticky-toolbar is active/visible.
+         */
+        .ck.ck-sticky-panel {
+            .ck.ck-sticky-panel__content_sticky {
+                top: calc($toolbarHeight);
+            }
+        }
```

Comment:
@danrot This is perhaps not necessary anymore, as I reduced height of editor. If we should use this feature, we should take the snackbar into account (only if it's visible) and also scrolling. But it would need JS to work as far as I could see.
Should I invest further here?

Reply:
That's not the only issue here, if the sticky header appears, and you scroll up again, the position does not change correctly anymore (the toolbar will stay right below the toolbar and have a huge gap to the text editor).

I think we could argue that if the height never exceeds the height of the screen without the toolbar anyway, we don't need this feature anyway... Could it be deactivated in some way? Maybe even just with a few lines of CSS? Would be my preferred solution, because it will result in less trouble.

@chirimoya WDYT?